ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -809,7 +809,9 @@ import '../nuxeo-button-styles.js';
       tableSettings.columns = {};
       if (this.columns) {
         this.columns.forEach((column, idx) => {
-          tableSettings.columns[column.field ? column.field : `col-${idx}`] = { hidden: column.hidden };
+          tableSettings.columns[column.field ? column.field : `col-${idx}`] = {
+            hidden: column.hidden,
+          };
```

Comment:
This kind of change can be on a separate commit for formatting and/or cleanup. It helps the reviewers. 😉

Reply:
I agree. As a matter o fact, unless this was fixed by our linting tools, I guess we don't even need to touch this.

Reply:
thanks! reverted that change!